src/shared/config.js

```diff
@@ -9,4 +9,7 @@ export const badgerBrain = {
 export const imageAssetsEndpoint =
   '//res.cloudinary.com/red-badger-assets/image/upload/events/';
 
+export const imageMasterAssetsEndpoint =
```

Comment:
i'll remove this for now - this was part of the idea to have transformations enabled on Cloudinary